tests/test_sample_rayservice_yamls.py

```diff
@@ -323,24 +338,23 @@ def test_service_autoscaling(self, set_up_cluster):
         scale_down_rule = AutoscaleRule(
             query={"path": "/signal", "json_args": {}},
             num_repeat=1,
-            expected_worker_pods=1,
+            expected_worker_pods=0,
             timeout=400,
             message="Releasing all blocked requests. Worker pods should start scaling down..."
         )
         cr_events: List[CREvent] = [
             RayServiceAddCREvent(
-                custom_resource_object=self.cr,
```

Comment:
Without this PR, the `custom_resource_object` in RayServiceAddCREvent is sourced from `ray_v1alpha1_rayservice.yaml`, and the `filepath` is set to `ray-service.autoscaler.yaml`. This is definitely a bug.